cmd/itzo-launcher/itzo-launcher.go

```diff
@@ -3,16 +3,17 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/elotl/itzo-launcher/pkg/cloudinit"
```

Comment:
Nitpicking: did your IDE move this import?

Reply:
Yes, nice catch! That's pretty strange ;)